containers/wing.toml

```diff
@@ -0,0 +1,2 @@
+[containers]
+provider="eks"
```

Comment:
Isn't this basically what a platform is? Having this as a parameter is kinda confusing in our current model

Reply:
I agree but as long as we don't have a way to inject dependencies for non-SDK classes (https://github.com/winglang/wing/issues/4930) we need something like this in order to be able to configure the behavior of `containers.Workload`.